content-services/latest/install/containers/docker-compose.md

```diff
@@ -397,7 +397,7 @@ The Docker Compose file provides some default configuration. This section lists
 | FILE_STORE_URL | Shared file store URL (in this case, the name of the container is used) |
 | TRANSFORM_ENGINE_REQUEST_QUEUE | Name of the queue. The default value is `org.alfresco.transform.engine.aio.acs` |
 | PDFRENDERER_EXE | Location of the PDF Renderer binary. The default value is `/usr/bin/alfresco-pdf-renderer` |
-| LIBREOFFICE_HOME | Location of the LibreOffice installation. The default value is `/opt/libreoffice7.2` |
```

Comment:
Are we supporting LibreOffice 7.3 now, or is this the result of a mass find/replace?

Reply:
That should still support LibreOffice 7.2